extension/package.json

```diff
@@ -216,6 +216,11 @@
         "command": "dvc.init",
```

Comment:
Do we want a more clear message? Like "Killed experiments EXP_NAME and EXP_NAME" for example 

<img width="486" alt="Screenshot 2023-01-10 at 8 47 37 AM" src="https://user-images.githubusercontent.com/43496356/211582639-6a8990da-334a-42e1-8eab-9287839f97ae.png">

Reply:
We just pass on whatever the CLI provides. Operation successful is what we show when there is no output.

Reply:
For more context: If the DVC team decided to add output for the command and we have added our own custom output then we will have probably diverged forever. IMO it is better to not "paper over the cracks" for things like this.

Reply:
Let's create a ticket / PR to update this on the DVC side with this screenshot and motivation please. Good catch @julieg18 ! Good point @mattseddon on diverging.